benchmarks/newton_solver_benchmark_set/nonlinear_channel_flow/input_t.prm

```diff
@@ -11,13 +11,16 @@ set Start time                             = 0
 set Adiabatic surface temperature          = 0
 set Surface pressure                       = 0
 set Use years in output instead of seconds = false  # default: true
-set Nonlinear solver scheme = iterated IMPES #Newton Stokes
+set Nonlinear solver scheme = iterated Advection and Stokes #Newton Stokes
```

Comment:
Is `#Newton Stokes` there in case someone wants to easily switch the solver to `iterated Advection and Newton Stokes`?

Reply:
The above comment is not necessarily pertinent to this PR.

Reply:
I think so? I don't mind it being there. This change just changes the deprecated name for the scheme.